pkg/util/cluster/cluster.go

```diff
@@ -146,6 +146,13 @@ func (c *Cluster) Create(ctx context.Context, vnetResourceGroup, clusterName str
 		visibility = api.VisibilityPrivate
 	}
 
+	var fipsEnabled bool
+	if os.Getenv("ARO_FIPS") != "" {
+		fipsEnabled = true
+	} else {
+		fipsEnabled = false
+	}
```

Comment:
I think with the new features we just tend to enable them by deafult in `./hack/cluster create` and CI (this is where we use `pkg/util/cluster`). For example, Roland is working on enabling disk encryption by default in https://github.com/Azure/ARO-RP/pull/1681.

But long term we needa better approach for CI: we need to test different feature combinations.

Reply:
I think - enable by default here.